### DISCUSSION TITLE
Prometheus Remote Write to Victoria Metrics

### DISCUSSION THREAD

**Discussion by Manniac07** [vectordotdev/vector#18614] [Q&A]
I am working on reading csv's from a directory, transforming them into metrics, and sending them to Victoria Metrics. Everything works fine until I need to drop multiple csv files into that directory. (This would be a valid scenario since the files are being ftp'd from an Oracle Database and there is always a possibility of files getting backed up.) When I try to drop multiple files, I am getting duplicate values and less than the number of files dropped. For instance, I tried with 10 files, I see 5 unique values and 3 duplicates. In Victoria Metrics, the values I am expecting from the 10 csv's are:

Client - 953
Client - 1401
Client - 1158 
Client - 1278...

Source:
```
[source.files]
 type = "file"
 glob_minimum_cooldown_ms = 45000
 include = ["directory/*csv"]
 oldest_first = true
 read_from = "beginning"

```
Sink:
```
[sink.console]
 type = "prometheus_remote_write"
 inputs = ["my input from transform"]
 endpoints = "https://hkljlkjh.com/insert/"
 [sinks.console.request]
   rate_limit_duration_sec = 10
   rate_limit_num = 15
 [sinks.console.buffer]
    type = "memory"
    max_events = 10000
```

**Comment by jszwedko** [comment]
You might already be aware, but note that `read_from` only applies to _new_ files. If Vector already read some of the other files it will have checkpointed them and will only read new data written rather than the whole file again. You can clear out the checkpoints in the data directory if you want to reset and read from the start again.

**Reply by Manniac07** [reply]
We enabled ignore checkpoints and the same results. On Victoria Metrics side we enabled dedup to 0s and still getting same result

**Comment by dsmith3197** [comment]
Could you post your full config and some example data? We don't have enough information to help troubleshoot this issue.
